source/documentation/using_the_api/getting_updates.md

```diff
@@ -1,20 +1,97 @@
 ## Getting updates
 
-You can find the latest entry number by looking at the register information (use the [`GET /register` endpoint](#getregister)) and comparing the most recent entry number with your own copy.
+You can use [the `GET /download-register/` endpoint](#download) to download a full new copy of a register.
 
-Example request: `https://local-authority-eng.register.gov.uk/register`
+### See how many updates have been made to a register
 
-Download a full new copy of the register by using [the `GET /download-register` endpoint](#download).
+You can use the [`GET /records/` endpoint](#getrecord) to return all the records from a register. In each record, there is an [`entry-number` property](#entries). The largest value for this is the current size of a register. You can compare this with a local copy. 
```

Comment:
I think the preferred way to know the latest (biggest) entry number is via `GET /register/` so probably I would flip this paragraph with the next one and arguably I would remove the suggestion about finding the largest value. It can be an expensive operation for large registers.

Regarding comparing with the local copy, could you elaborate a bit more what is the user expected to do and to get out of it?

Reply:
Another important bit is: if you got into the trouble to get all records, there is no real need to compare against anything; a full replace will suffice.